report_pre_thesis/src/index.tex

```diff
@@ -73,7 +73,9 @@ \section{Related work}
 % \subsection{comment-end}
 
 \section{Definition}
-Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS. Additionally, let $S$ be the set of all possible spectra emitted from these compositions.
+Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS.
+
+Let $S$ be the set of all possible spectra, where each spectrum is a multidimensional vector in $\mathbb{R}^n$ where $n$ being the number of wavelengths captured by the LIBS spectrometer. For any spectrum $s_i$, it holds that \(s_i \in S \).
```

Comment:
```suggestion
Let $S$ be the set of all possible spectra, where each spectrum $\mathbf{s} \in S$ is a multidimensional vector in $\mathbb{R}^n$ with $n$ being the number of wavelengths captured by the LIBS spectrometer for a single laser pulse. $\mathbf{s}$ holds the intensities of light emitted from the plasma that is created from the laser ablation process.
```

Likely closer, no?

Reply:
So you can see it rendered:

Let $S$ be the set of all possible spectra, where each spectrum $\mathbf{s}\_{i} \in S$ is a multidimensional vector in $\mathbb{R}^{n}$ with $n$ being the number of wavelengths captured by the LIBS spectrometer for a single laser pulse. $\mathbf{s}_{i}$ holds the intensities of light emitted from the plasma that is created from the laser ablation process.

Reply:
Bolded $\mathbf{s}$ because conventionally vectors are lowercase bold.